phpstan-baseline.neon

```diff
@@ -507,6 +507,16 @@ parameters:
 			count: 1
 			path: tests/Rules/Operators/OperandsInArithmeticSubtractionRuleTest.php
 
+		-
+			message: "#^Class PHPStan\\\\Rules\\\\Operators\\\\OperandsInTernaryOperatorRuleTest extends generic class PHPStan\\\\Testing\\\\RuleTestCase but does not specify its types\\: TRule$#"
```

Comment:
I really don' like to add new stuff to the baseline, it should be done correctly instead.